telemetry/main.schema.json

```diff
@@ -846,13 +846,16 @@
           "minimum": 0
         },
         {
-          "type": "string"
+          "type": "string",
+          "pattern": "^[a-zA-Z][a-zA-Z0-9_.]*[a-zA-Z0-9]$"
```

Comment:
The build process updates the schemas/telemetry directory (with a beautified, sorted output) and yes those files should be included as part of the PR.